app/src/main/res/layout/fragment_counter.xml

```diff
@@ -23,11 +23,60 @@
         android:id="@+id/message"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:layout_marginTop="24dp"
+        android:layout_marginTop="16dp"
         android:textAppearance="@style/TextAppearance.ArchDemo.Title3"
         tools:text="Some message here"
         app:layout_constraintTop_toBottomOf="@id/title"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent" />
 
+    <TextView
+        android:id="@+id/click_count_title"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="48dp"
+        android:textAppearance="@style/TextAppearance.ArchDemo.Regular"
+        android:text="Click count: "
```

Comment:
Would you mind extracting those to String resources?
Not sure how far you want to go, but even formatted string could be shown here:
```<string name="click_count">Click count: %d</string>```

Reply:
Resources are out-of-scope of this lecture.